src/main/js/auth/login-container.js

```diff
@@ -6,42 +6,43 @@
 
 import React from "react";
 import Login from "./login";
-import { useNavigate } from "react-router";
+import {useNavigate} from "react-router";
 import TrackrUserService from "../user/trackr-user-service";
 
 const LoginContainer = (props) => {
-  const service = new TrackrUserService();
-  const navigate = useNavigate();
+    const service = new TrackrUserService();
+    const navigate = useNavigate();
 
-  /**
-   * Handle submit of the login form by authenticating with the backend.
-   *
-   * @param credentials Login form values
-   */
-  const handleLoginSubmit = (credentials) => {
-    service
-      .login(credentials)
-      .then(function (response) {
-        TrackrUserService.authenticate();
-        navigate("/dashboard", { replace: true });
-        props.setAlert({
-          show: true,
-          variant: "success",
-          message: "Successfully authenticated!",
-        });
-        setTimeout(() => props.setAlert({ show: false }), 2000);
-      })
-      .catch(function (error) {
-        props.setAlert({
-          show: true,
-          variant: "danger",
-          message: `${error.response.data.message}`,
-        });
-        setTimeout(() => props.setAlert({ show: false }), 2000);
-      });
-  };
+    /**
+     * Handle submit of the login form by authenticating with the backend.
+     *
+     * @param credentials Login form values
+     */
+    const handleLoginSubmit = (credentials) => {
+        service
+            .login(credentials)
+            .then(function (response) {
+                TrackrUserService.authenticate();
+                sessionStorage.setItem("trackrToken", response.data.additionalData[0].jwtToken);
```

Comment:
`sessionStorage.setItem("trackrToken", response.data.additionalData[0].jwtToken);`
We need to remove as we don't want JS accessing the JWT to prevent XSS attack.

Reply:
done